openedx/features/course_experience/utils.py

```diff
@@ -82,3 +82,33 @@ def mark_last_accessed(user, course_key, block):
     mark_last_accessed(user, course_key, course_outline_root_block)
 
     return course_outline_root_block
+
+
+def get_flat_course_structure(course_id):
+    """
+    Get course structure directly from modulestore.
+
+    Args:
+        course_id (str): course id as a string
+
+    Returns:
+        List with dicts containing names of course sections and subsections
+    """
+    course_key = CourseKey.from_string(course_id)
+    root_block = modulestore().get_course(course_key)
+
+    course_structure = []
+    sections = root_block.get_children()
+
+    if sections:
```

Comment:
get_children() always returns a list object, so if condition here is redundant

Reply:
fixed, thanks